www/js/controller.js

```diff
@@ -0,0 +1,4 @@
+/* This simple app has only one view, and so only one controller.
+ * Its job is to provide data (from the weatherData service) for display
+ * by the html page (index.html).
+ */
```

Comment:
I'm not sure what this file is for. Haha. You can move your angular controller into it.